packages/bot-skeleton/src/scratch/hooks/gesture.js

```diff
@@ -27,6 +27,9 @@ Blockly.Gesture.prototype.updateIsDraggingFromFlyout = function () {
         this.startBlock = null;
         this.targetBlock = this.flyout.createBlock(this.mostRecentEvent, this.targetBlock);
         this.targetBlock.select();
+
+        // retuning false since block has been dragged from flyout
+        Blockly.Gesture.inProgress = () => false;
```

Comment:
@rupato-deriv I think it shouldn't be inside `updateIsDraggingFromFlyout` as the drag continues. Can we check if there is any event that fires when drag is done/completed?

Reply:
done ✅ changed and used block current workspace method

Reply:
@rupato-deriv this code is still here 😄

Reply:
removed so fast bhai